src/validate/format.rs

```diff
@@ -94,4 +94,5 @@ pub struct FileAttr {
     pub uri: String,
     #[serde(rename = "type")]
     pub file_type: String,
+    pub cdn: Option<bool>,
 }
```

Comment:
I think it will be better to leave this as a boolean, and skip serializing if it is false.

```suggestion
    #[serde(default, skip_serializing_if = "bool_is_false")]
    pub cdn: bool,
}

fn bool_is_false(value: &bool) -> bool {
    !value
}
```

Reply:
@KartikSoneji  does that lead to unexpected behavior from the perspective of the user of the cli? In the case where they set `cdn: false` and then see that its striped from the metadata json. 

The optional already handles serializations when the cdn field is not present but in the case of someone setting it why not respect their configuration?

Reply:
I would argue that is a desirable feature, since cdn defaults to false and removing it would result it smaller metadata without redundant properties.

Reply:
What's the desired outcome if the user doesn't specify it? The `Option<bool>` solution results in a `null` value whereas skipping serializing it will result in no `cdn` attribute listed in the metadata.

Reply:
E.g. metadata with no CDN specified: https://solscan.io/token/9KqVASYiLjGX76bNJ11MVUvHmTzsqouE1cJyuqnFHQ3b?cluster=devnet#metadata

Reply:
If the user doesn't specify it, or if it is false, I think it should not be included since it only serves as clutter.
The only time `"cdn": true` should be in the json is if the user specifies it.

Reply:
@kespinola that seems consistent with the example JSON files I've seen. Do you agree?